# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gdmap/04-pixmap-location.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- gdmap-v1.3.1/src/about.c.vanilla	2025-06-28 11:07:02.354338904 +0200
+++ gdmap-v1.3.1/src/about.c	2025-06-28 11:07:53.798752067 +0200
@@ -24,7 +24,7 @@
 #ifdef PACKAGE_SOURCE_DIR
     temp = g_strdup_printf("%s/data/%s", PACKAGE_SOURCE_DIR, filename);
 #else
-    temp = g_strdup_printf("%s/pixmaps/%s", PACKAGE_DATA_DIR, filename);
+    temp = g_strdup_printf("/opt/gnome/%s/pixmaps", PACKAGE_DATA_DIR, filename);
 #endif
     pix = gdk_pixbuf_new_from_file(temp, NULL);
     g_free(temp);
--- gdmap-v1.3.1/src/gui_main.c.vanilla	2025-06-28 11:08:01.687664539 +0200
+++ gdmap-v1.3.1/src/gui_main.c	2025-06-28 11:08:09.181775198 +0200
@@ -886,7 +886,7 @@
 #ifdef PACKAGE_SOURCE_DIR
     temp = g_strdup_printf("%s/data/%s", PACKAGE_SOURCE_DIR, filename);
 #else
-    temp = g_strdup_printf("%s/pixmaps/%s", PACKAGE_DATA_DIR, filename);
+    temp = g_strdup_printf("/opt/gnome/%s/pixmaps", PACKAGE_DATA_DIR, filename);
 #endif
     pix = gdk_pixbuf_new_from_file(temp, NULL);
     g_free(temp);
